client/state/index.js

```diff
@@ -111,7 +112,9 @@ export const reducer = combineReducers( {
 	ui,
 	users,
 	wordads,
-} );
+};
+
+export const reducer = combineReducers( { ...extensions.reducers(), ...reducers } );
```

Comment:
It should be impossible as the names of the extensions are derived from the listing of directories under `client/extensions`